tests_e2e.py

```diff
@@ -52,13 +54,13 @@ def request(method, path, headers={}, body='', cookies=None):
 
     if response.history and response.history[0]:
         # This code branch will be executed if there is a redirect
-        ret['code']    = response.history[0].status_code
```

Comment:
This is just an example, but please refrain from making these styling changes on code parts unrelated to your PR. This makes it hard to review.

Reply:
Yeh I think VSCode automatically formats the file. I'll go back to those files and revert the changes to make the pull request cleaner 👍